setup.cfg

```diff
@@ -0,0 +1,71 @@
+[metadata]
+name = OMMBV
+version = file: OMMBV/version.txt
+url = https://github.com/rstoneback/OMMBV
+author = Russell A. Stoneback
+author_email = russell@stoneris.com
+description = 'Orthogonal geomagnetic vector basis and field-line mapping for multipole magnetic fields.'
+long_description = file: README.md, CHANGELOG.md
+long_description_content_type = text/markdown
+classifiers =
+    Development Status :: 5 - Production/Stable
+    Topic :: Scientific/Engineering :: Physics
+    Intended Audience :: Science/Research
+    License :: OSI Approved :: BSD
+    Natural Language :: English
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+    Operating System :: MacOS :: MacOS X
+    Operating System :: POSIX :: Linux
```

Comment:
Update based on which tests are run

Reply:
I originally left Mac on there since I'm running local tests. I can update if the standard is to report what is publicly run.

Reply:
Updated.